packages/template-retail-react-app/app/pages/product-detail/index.test.js

```diff
@@ -160,3 +161,90 @@ describe('product set', () => {
         })
     })
 })
+
+describe('product bundles', () => {
+    beforeEach(() => {
+        global.server.use(
+            // Use product bundle instead of product set
+            rest.get('*/products/:productId', (req, res, ctx) => {
+                return res(ctx.delay(0), ctx.status(200), ctx.json(mockProductBundle))
+            }),
+            // For adding items to basket
+            rest.post('*/baskets/:basketId/items', (req, res, ctx) => {
+                return res(ctx.json(basketWithProductSet))
+            })
+        )
+    })
+
+    test('renders multi-product layout', async () => {
+        renderWithProviders(<MockedComponent />)
+
+        await waitFor(
+            () => {
+                expect(screen.getAllByTestId('product-view')).toHaveLength(4) // 1 parent + 3 children
+            },
+            {timeout: 5000}
```

Comment:
Are these timeouts necessary on our local machines? Or are they intended for the slower CI environments?

If for CI, these `waitFor` timeouts of 5000 are no longer needed, since we've already set the default to 6000 here globally: https://github.com/SalesforceCommerceCloud/pwa-kit/blob/b7c0aafcdd5eb88e986db039b82bcbca281454ff/packages/template-retail-react-app/jest-setup.js#L27-L30